static/src/styles/landing/LandingAbout.styl

```diff
@@ -1,17 +1,22 @@
-.LandingAbout {
+.LandingAbout__section {
   padding: grid(4);
 }
 
 .LandingAbout__imageContainer {
   flex: 1;
 }
 
+.LandingAbout__section:nth-child(even) .LandingAbout__imageContainer {
+  order: 10;
```

Comment:
While this isn't in effect in mobile views as display isn't flexbox, do you think it would better describe intent if this declaration were in the media query below?

At first when reading this, I was confused as I thought there would be alternating images in our mobile view - I had to think through it to figure out what was actually going on.

Reply:
@Pinpickle I think you're right - should be fairly trivial to change too 😄